src/video/wayland/SDL_waylandmouse.c

```diff
@@ -592,7 +592,16 @@ static SDL_Cursor *Wayland_CreateSystemCursor(SDL_SystemCursor id)
 
 static SDL_Cursor *Wayland_CreateDefaultCursor(void)
 {
-    return Wayland_CreateSystemCursor(SDL_SYSTEM_CURSOR_DEFAULT);
+    // return Wayland_CreateSystemCursor(SDL_SYSTEM_CURSOR_DEFAULT);
+    SDL_SystemCursor id = SDL_SYSTEM_CURSOR_DEFAULT;
```

Comment:
It might be worth moving this into SDL_mouse.c and calling it from there instead of duplicating this logic everywhere.

Reply:
It shouldn't, no, but you might need to include the SDL_mouse_c.h to see the function.